### DISCUSSION TITLE
Add Chinese answers

### DISCUSSION THREAD

**Discussion by GitHub-Of-WangQiang** [community/community#72859] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Although I can enter a command to ask the co-pilot to answer in Chinese, it is always a little inconvenient. Can I directly add a language selection function so that people with different languages can directly see the answers in their native language? Also, it is best to ask questions in your native language rather than English

**Comment by jasonwang82** [comment]
It is also slow for chinese chat
